057-AzureRouteServer/Coach/README.md

```diff
@@ -0,0 +1,72 @@
+
+## Coach's Guides
+
+- Challenge 00: **[Prerequisites - Ready, Set, GO!](./Solution-00.md)**
+	 - Prepare your workstation to work with Azure.
+- Challenge 01: **[Building a Basic Hub and Spoke Topology utilizing a central Network Virtual Appliance](./Solution-01.md)**
+	 - Warming up
+- Challenge 02: **[Introduce Azure Route Server and peer with a Network Virtual appliance](./Solution-02.md)**
+	 - Azure Route Server basic concepts, interaction with NVAs
+- Challenge 03: **[Connect Network Virtual Appliance to an SDWAN environment](./Solution-03.md)**
+	 - Azure Route Server interaction with Virtual Network Gateways (VPN or ExpressRoute)
+- Challenge 04: **[Introduce a High Availability with Central Network Virtual Appliances](./Solution-04.md)**
+	 - High Availability concepts
+
+## Coach Prerequisites
+
+This hack has pre-reqs that a coach is responsible for understanding and/or setting up BEFORE hosting an event. Please review the [What The Hack Hosting Guide](https://aka.ms/wthhost) for information on how to host a hack event.
+
+The guide covers the common preparation steps a coach needs to do before any What The Hack event, including how to properly configure Microsoft Teams.
+
+### Student Resources
+
+Before the hack, it is the Coach's responsibility to download and package up the contents of the `/Student/Resources` folder of this hack into a "Resources.zip" file. The coach should then provide a copy of the `Resources.zip` file to all students at the start of the hack. 
+
+In Challenge 00, the students will run a script from this Resources.zip file that deploys a baseline Hub & Spoke network topology into their Azure subscriptions with Linux VMs.  
+
+The coach should provide the students with the Username and password used for the Ubuntu VMs:
+- username: azureuser
+- psswd: Msft123Msft123
+
+**NOTE:** The script deploys Active/Active VPNs with BGP and the correspondent Vnet Peering attributes for transitivity. However, other aspects such as configuring Local Network Gateways, setting up required Route Tables (UDRs) will need to be done manually. Simulated on-premises and Central NVA templates are provided separately throughout the challenge
+
+Always refer students to the [What The Hack website](https://aka.ms/wth) for the student guide: [https://aka.ms/wth](https://aka.ms/wth)
+
+**NOTE:** Students should **not** be given a link to the What The Hack repo before or during a hack. The student guide does **NOT** have any links to the Coach's guide or the What The Hack repo on GitHub.
+
+### Additional Coach Prerequisites
+
+The coach should offer the student the aforementioned script file at the start of the event so credits using the VPN Gateway are not consumed before the Hack. If the student opts to do the initial setup manually, bring awareness that this is going to consume time and is prone to errors. 
```

Comment:
Not sure what the point of this paragraph is?  Why would the student be deploying the script before the coach starts the hack?  Is manual set up of the baseline H&S topology supported/recommended as an option for students?

These notes, if relevant, belong in the C0 coach guide.  This section can be omitted if there are not additional pre-reqs for a coach to do other than create the Resources.zip file and provide it to the students.

Reply:
In the last conversation we had we agreed that students might benefit more from applying the script while the lecture happens since it takes a long time to deploy a VPN Gateway individually and deploying the base topology can take up to half a day and is not the main point of the hack.

Reply:
Edited after discussion with @maurojasmsft